clang/test/CIR/CodeGen/multi-vtable.cpp

```diff
@@ -86,7 +86,7 @@ int main() {
 // LLVM-DAG: @_ZTVN10__cxxabiv117__class_type_infoE = external global ptr
 
 //   typeinfo name for Mother
-// CIR: cir.global linkonce_odr @_ZTS6Mother = #cir.const_array<"6Mother" : !cir.array<!s8i x 7>> : !cir.array<!s8i x 7> {alignment = 1 : i64}
+// CIR: cir.global linkonce_odr comdat @_ZTS6Mother = #cir.const_array<"6Mother" : !cir.array<!s8i x 7>> : !cir.array<!s8i x 7> {alignment = 1 : i64}
```

Comment:
There is change in CIR but I don't see anything in LLVM, what's the deal?

Reply:
> There is change in CIR but I don't see anything in LLVM, what's the deal?

great point!  I wasn't sure if it is useful for us to lower comdat to LLVM IR at this point.  One thing for sure is that we'd need to add a new pass to CIR LLVM lowering. After all, there is already a pass LLVMAddComdats who does similar thing. I just updated with a commit that illustrates the idea(the commit is for info only) . (If decide we do want such a pass, I'll revert the commit, and add our own CIR addComdats pass)
Let me know if we should add such pass or not bother for now.

Reply:
The LLVM we emit should be the same that the OG emits, how is it for this test case and others? It's fine if you add that in another PR though, it just needs to be explicit mentioned in the description (or mention somehow to me).

Reply:
Actually, it was in the original PR description that we will leave LLVM lowering to future PR, but now I found its actually not much work to implement it. so for this PR I implemented it for GlobalOp, if things look good, we will do the same lowering implementation for FuncOp.